.github/workflows/external-services-ip-sets-sync.yml

```
@@ -0,0 +1,50 @@
+name: External Services WAF IP Sets Sync
+
+on:
+  workflow_dispatch: # Allow manual trigger
```

I think you could add this just for testing, so we don't have to merge this to main before we see if it fails for one reason or another while executing on the GHA runner. Once we have a successful build we can take it out and merge to main. Did we ever specify a schedule this should run on as well? Or will we be manually running this like the AB2D IP set sync?

```suggestion
on:
  workflow_dispatch: # Allow manual trigger
  push:
    branches:
      - jscott/PLT-506
```

---

And just now seeing the runs from this in test, so disregard this suggestion